<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>460</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>LAIN MEDIA FILE PARSER</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabImages">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <attribute name="title">
      <string>Images</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>271</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAudio">
     <attribute name="title">
      <string>Audio</string>
     </attribute>
     <widget class="QListWidget" name="audioList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>271</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabVideos">
     <attribute name="title">
      <string>Videos</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>271</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDocs">
     <attribute name="title">
      <string>Docs</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>271</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="parsingAnimation">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>401</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>160</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="switchesGroup"/>
   </widget>
   <widget class="QLabel" name="btnSave">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="btnSettings">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Enter url - site address for parsing</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
